tests/general/KeywordBearTest.py

```diff
@@ -214,3 +214,20 @@ def test_keyword_regex(self):
             self.assertEqual(result[0].message, 'The line contains the keyword'
                                                 " 'Issue #123' which resulted "
                                                 'in a match with given regex.')
+
+    def test_wrong_language(self):
+        section = Section('')
+        section.append(Setting('language', 'anything'))
+        section.append(Setting('keywords', 'TODO'))
+        uut = KeywordBear(section, Queue())
+        dep_results = {
+            'AnnotationBear': HiddenResult(
+                'AnnotationBear',
+                'coalang specification for anything not found.')
+        }
+
+        text = ['a==b']
```

Comment:
Thanks @sils for your valuable suggestions.  Actually I had wrong concepts about working of Keyword Bear. I was therefore completely pissed off. Well a question for now. Does Keyword Bear have to return the position of  keywords in comments only or it returns position of keywords in the code too?